
device at starting_train:  cuda
/usr/local/lib/python3.7/dist-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
current acc: tensor(0.1016, device='cuda:0')
/content/projects-skeleton-code/networks/StartingNetwork.py:35: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  x = F.softmax(x)
current acc: tensor(0.5365, device='cuda:0')
current acc: tensor(0.5895, device='cuda:0')
current acc: tensor(0.5913, device='cuda:0')
current acc: tensor(0.5960, device='cuda:0')
Epoch: 1  Train Loss: 1.2965  Train Acc: 60.0260
Time Elapsed : 93.5116s
device at starting_train:  cuda
current acc: tensor(0.6328, device='cuda:0')
current acc: tensor(0.6250, device='cuda:0')
current acc: tensor(0.6072, device='cuda:0')
current acc: tensor(0.6099, device='cuda:0')
current acc: tensor(0.6034, device='cuda:0')
Traceback (most recent call last):
  File "main.py", line 65, in <module>
    main(config)
  File "main.py", line 50, in main
    num_epochs=config['epochs'],
  File "/content/projects-skeleton-code/train_functions/starting_train.py", line 158, in starting_train
    loss_valid, acc_valid = validate(network, epoch, criterion, valid_loader, device = device)
TypeError: validate() got an unexpected keyword argument 'device'